apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -30,11 +30,18 @@ export function useApproveCallback(
   options: {
     addToTransaction
     targetAmount?: bigint
+
+    /**
+     * Use paymaster if available.
+     * Enable only if Gas Token Selector is present on the interface.
+     */
+    enablePaymaster?: boolean
   } = {
     addToTransaction: true,
+    enablePaymaster: false,
   },
```

Comment:
better set the default value one by one
```suggestion
  {
    enablePaymaster = false,
  }: {
    addToTransaction?: boolean
    targetAmount?: bigint
    enablePaymaster?: boolean
  } = {}
```